### PR TITLE
implement display for criterion

### PR DESCRIPTION
**Pull request by MarinPostma**
Implement `Display` for `Criterion`.

Needed to return the criterion with the correct formating to the user.

### CODE REVIEW

File: milli/src/criterion.rs [hunk 1]
```diff
@@ -63,3 +64,20 @@ pub fn default_criteria() -> Vec<Criterion> {
...
+
+        match self {
+            Typo            => write!(f, "typo"),
+            Words           => write!(f, "words"),
+            Proximity       => write!(f, "proximity"),
+            Attribute       => write!(f, "attribute"),
+            WordsPosition   => write!(f, "wordsPosition"),
+            Exactness       => write!(f, "exactness"),
+            Asc(attr)       => write!(f, "asc({:?})", attr),
+            Desc(attr)      => write!(f, "desc({:?})", attr),
```
Comment (Kerollmops) [inline milli/src/criterion.rs:80]:
```suggestion
            Typo            => f.write_str("typo"),
            Words           => f.write_str("words"),
            Proximity       => f.write_str("proximity"),
            Attribute       => f.write_str("attribute"),
            WordsPosition   => f.write_str("wordsPosition"),
            Exactness       => f.write_str("exactness"),
            Asc(attr)       => write!(f, "asc({:?})", attr),
            Desc(attr)      => write!(f, "desc({:?})", attr),
```

**Review by Kerollmops — CHANGES_REQUESTED** [review]

File: milli/src/criterion.rs (hunk 1 above)
Comment (MarinPostma) [reply milli/src/criterion.rs:80]: I don't mind changing, but why? I find it somewhat nicer when they are all the same :)

Comment (Kerollmops) [reply milli/src/criterion.rs:80]: Just to skip the formatting part, as it is not necessary :)

File: milli/src/criterion.rs [hunk 2]
```diff
@@ -63,3 +64,20 @@ pub fn default_criteria() -> Vec<Criterion> {
...
+
+        match self {
+            Typo            => f.write_str("typo"),
+            Words           => f.write_str("words"),
+            Proximity       => f.write_str("proximity"),
+            Attribute       => f.write_str("attribute"),
+            WordsPosition   => f.write_str("wordsPosition"),
+            Exactness       => f.write_str("exactness"),
+            Asc(attr)       => write!(f, "asc({:?})", attr),
+            Desc(attr)      => write!(f, "desc({:?})", attr),
```
Comment (Kerollmops) [inline milli/src/criterion.rs:80]:
```suggestion
            Asc(attr)       => write!(f, "asc({})", attr),
            Desc(attr)      => write!(f, "desc({})", attr),
```

**Review by Kerollmops — CHANGES_REQUESTED** [review]

**Pull request merged**